src/main/java/pl/touk/parking/domain/ParkingUsage.java

```diff
@@ -0,0 +1,70 @@
+package pl.touk.parking.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import java.time.Clock;
+import java.time.LocalDateTime;
+
+import static java.time.temporal.ChronoUnit.HOURS;
+import static javax.persistence.CascadeType.PERSIST;
+
+@Entity
+public class ParkingUsage {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @OneToOne(cascade = PERSIST)
+    private Driver driver;
+    private LocalDateTime startTime;
```

Comment:
a jak ktoś będzie parkował przez pół godziny w momencie zmiany czasu z letniego na zimowy?

Reply:
Dobre! Nie pomyślałem o tym :)
W takim razie powinienem użyć zegara UTC, albo uwzględniać w obliczeniach strefę czasową. No i dodać test na to.